web/src/main/java/org/mskcc/oncotree/api/ApiOriginFilter.java

```diff
@@ -1,27 +0,0 @@
-package org.mskcc.oncotree.api;
```

Comment:
We may want a filter for the resetCache endpoint (only accepting requests from inside the MSKCC network). We should talk this over.